commands/dkcheck.ml

```diff
@@ -1,141 +1,181 @@
 open Kernel
 open Api
-open Basic
-
-let _ =
-  let run_on_stdin = ref None in
-  let export = ref false in
-  let beautify = ref false in
-  let quiet = ref false in
-  let deprecated old_flag new_flag spec =
-    let warning () =
-      Debug.(debug d_warn)
-        "[DEPRECATED] Flag %s is deprecated ! Use %s instead.@." old_flag
-        new_flag
-    in
-    (old_flag, Arg.Tuple [Arg.Unit warning; spec], "")
+open Cmdliner
+
+(** Common options *)
+type config = {no_color : bool; debug : string; quiet : bool; verbose : bool}
```

Comment:
It seems `quiet` and `verbose` are conflictual. What would it means to have `quiet=true` and `verbose=true`?

The doc says `verbose` overrides `quiet`. I think we do not need to be retrocompatible here and use only one flag to control the verbosity.